index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="navbar">
+        <a href="#" class="logo">My Logo</a>
+        <nav><img class="union" src="./images/Union.png"  alt=""></nav>
+    </header>
+    <section class="introduction">
```

Comment:
- [ ] [OPTIONAL] Good job fixing the background image 😃 , Kindly you may fix the margins to look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it was [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![mobile01](https://user-images.githubusercontent.com/57408419/220439616-cca2107d-96ca-4383-9b6c-e5efb32ca5d0.jpg)| |![mobile](https://user-images.githubusercontent.com/57408419/220439687-06b46b08-d08b-405a-956b-4ad7899924e0.jpg) | |